.github/workflows/release.yml

```diff
@@ -1,24 +1,33 @@
 name: Release
 on:
-  release:
-    types:
-      - created
-
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to release (which will also be used for the tag)'
+        required: true
+        type: string
+permissions:
+  contents: write # To write the version change to the repository
 jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Setup Java
-        uses: actions/setup-java@v4
+      - name: Update version in gradle.properties
+        run: |
+          sed -i 's/^version=.*$/version=$VERSION/' gradle.properties
+          git config --local user.name "github-actions[bot]"
+          git config --local user.email "github-actions[bot]@users.noreply.github.com"
+          git add gradle.properties
+          git commit -m "Update version to $VERSION"
+          git tag -a $VERSION -m "Release $VERSION"
+          git push --tags
```

Comment:
Before pushing, the version should be bumped to "version with bugfix + 1-SNAPSHOT", and also commit that.

Reply:
so increas patch version by 1 and add snapshot?

Reply:
yes

Reply:
Added in https://github.com/Blazebit/blaze-query/pull/20/commits/4569785738eb2b2c2b1f55a8b382f71b872441ca

Reply:
and: https://github.com/Blazebit/blaze-query/pull/20/commits/cdc3a0a186a1a3633105b328d754cafc2f05d1bf

Reply:
Can you please do the version bumping as part of this workflow and also only do a single push? Otherwise there would be a period where the main branch is in an "invalid" state.

Reply:
Done